.travis.yml

```diff
@@ -3,10 +3,7 @@ language: ruby
 cache: bundler
 
 rvm:
-  - 2.4
-  - 2.5
-  - 2.6
-  - 2.7
+  - truffleruby-head
```

Comment:
The tests fail, and I don't see why we are replacing all Ruby versions with this one.